rubocop.yml

```diff
@@ -105,3 +107,12 @@ Style/YodaCondition:
 Layout/SpaceBeforeBlockBraces:
   EnforcedStyle: space
   EnforcedStyleForEmptyBraces: space
+
+RSpec/NotToNot:
+  EnforcedStyle: to_not
```

Comment:
One day, I'm going to get us to reverse this rule. But not today...

Reply:
I preferred `not_to` as well, but I was overruled for some reason 🤷‍♀️

Reply:
to be, or to not be. that is a question.